docs/admin/authentication.md

```diff
@@ -256,6 +256,16 @@ To enable the plugin, configure the following flags on the API server:
 | `--oidc-groups-claim` | JWT claim to use as the user's group. If the claim is present it must be an array of strings. | groups | No |
 | `--oidc-ca-file` | The path to the certificate for the CA that signed your identity provider's web certificate.  Defaults to the host's root CAs. | `/etc/kubernetes/ssl/kc-ca.pem` | No |
 
+If a claim other than `email` is chosen for `--oidc-username-claim`, the value
```

Comment:
include a note in the table for the --oidc-username-claim... they might not make it this far otherwise